tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -32,10 +32,10 @@
 
 
     <!--Microsoft.Data.SqlClient-->
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" Condition="'$(TargetFramework)' == 'net462'" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.0.19239.1" Condition="'$(TargetFramework)' == 'net462'" />
     <PackageReference Include="Microsoft.Data.SqlClient" Version="5.0.1" Condition="'$(TargetFramework)' == 'net471'" />
```

Comment:
Should we try a couple other versions in here as well?

Reply:
We could but I don't have a strong reference on what other versions we should test. Could you give a list ?

Reply:
In general I think covering as many major versions as possible is a good way to do it. It looks like there's a 2.1.5, a 3.1.1, and a 4.1.1.

Reply:
ok. I will update with these versions.